client/src/extension.ts

```diff
@@ -169,9 +172,41 @@ async function ExecYseopCliCommandWithKbDirectoryPath(cliPath: string, ...words:
         window.showErrorMessage('This command must be used from within a workspace folder.');
         return;
     }
-
-    const workspaceFolders = workspace.workspaceFolders;
-    ExecYseopCliCommand(cliPath, ...words, workspaceFolders[0].uri.fsPath);
+    const workspaceFirstDir = workspace.workspaceFolders[0].uri.fsPath;
+
+    // Get the list of possible KBs in the workspace (like in library projects)
```

Comment:
Write proper functions and this comment will become virtually useless xD

Reply:
That's not completely true, since it will be part of the documentation of the new function :p
It is clear that having a function dedicated to this part would make the code easier to read.
I'll do it.